solutions/usaco-860.mdx

```diff
@@ -13,6 +13,131 @@ author: Nathan Gong
 
 <LanguageSection>
 
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int n, k;
+const int MAX_ROWS = 100;
+const int MAX_COLS = 10;
+char board[MAX_ROWS][MAX_COLS];
+vector<vector<bool>> visited(MAX_ROWS, vector<bool>(MAX_COLS, false));
```

Comment:
wow can't believe you actually took my suggestion
but uh
```suggestion
vector<vector<bool>> visited(MAX_ROWS, vector<bool>(MAX_COLS));
```
the default value is false jsyk